Examples/Io/Performance/ActsExamples/Io/Performance/VertexPerformanceWriter.cpp

```diff
@@ -349,6 +363,14 @@ ActsExamples::ProcessCode ActsExamples::VertexPerformanceWriter::writeT(
       }
     }
 
+    int nTracksOnTruthVertex = 0;
+    for (const auto& particle : associatedTruthParticles) {
+      int priVtxId = particle.particleId().vertexPrimary();
+      if (priVtxId == maxOccurrenceId) {
+        ++nTracksOnTruthVertex;
+      }
+    }
+
```

Comment:
This should be equal to maxOccurrence. (But check that)

Reply:
this is actually counting something else